helix-core/src/test/java/org/apache/helix/task/TestTaskStateModelFactory.java

```diff
@@ -0,0 +1,95 @@
+package org.apache.helix.task;
+
```

Comment:
Apache license?

Reply:
Not sure what happened there... Added it. Thanks for catching it!